deployment/build-images.sh

```diff
@@ -72,6 +77,9 @@ fi
 DOCKERFILE_NODE=deployment/Dockerfile-node
 : "${NODE_IMAGE_NAME:=mpc-node}"
 
+DOCKERFILE_NODE_GCP=deployment/Dockerfile-node-gcp
+: "${NODE_GCP_IMAGE_NAME:=mpc-node-gcp}"
```

Comment:
Hmm I'm a bit torn on whether we should have a separate image tag for this or not. But sure, if we're maintaining different images for now it does make sense. Let's keep this for now and get rid of it in a subsequent release.